Jucer2Reprojucer/main.cpp

```diff
@@ -1387,14 +1387,17 @@ int main(int argc, char* argv[])
       }
 
       const auto isAudioPlugin = projectType == "audioplug";
+      const auto pluginFormats = juce::StringArray::fromTokens(
+        jucerProject.getProperty("pluginFormats").toString(), ",", {});
       const auto hasJuceAudioProcessorsModule =
         jucerProject.getChildWithName("MODULES")
           .getChildWithProperty("id", "juce_audio_processors")
           .isValid();
 
       const auto hasVst2Interface = jucerVersionAsTuple > Version{4, 2, 3};
-      const auto isVstAudioPlugin =
-        isAudioPlugin && bool{jucerProject.getProperty("buildVST")};
+      const auto isVstAudioPlugin = isAudioPlugin
+                                    && (bool{jucerProject.getProperty("buildVST")}
+                                        || pluginFormats.contains("buildVST"));
```

Comment:
I know you didn't ask for my code review but here it is anyway :)
It would be nice to DRY the strings here (and for the other formats)

Reply:
`hasProperty` can be used for more concise code instead of `getProperty`

Reply:
Probably makes more sense to check `pluginFormats` first to short-circuit most times, since that's where it would be for non-legacy jucer files

Reply:
> I know you didn't ask for my code review but here it is anyway :)

No problem at all. Reviews are always appreciated!


> It would be nice to DRY the strings here (and for the other formats)

Feel free to propose a concrete solution, since anything I could think of only adds more code 😅 


> `hasProperty` can be used for more concise code instead of `getProperty`

If the `"buildVST"` property is set to 0, `bool{getProperty("buildVST")}` will return `false`, while `hasProperty("buildVST")` will return `true`. So here, we can't use `hasProperty`.


> Probably makes more sense to check `pluginFormats` first to short-circuit most times, since that's where it would be for non-legacy jucer files

I totally agree! Addressed in 8460bc9c793a5a5ade961feec30b1103d346f344.

Reply:
> Feel free to propose a concrete solution, since anything I could think of only adds more code 😅
>
I suggest refactoring into a function which checks both, then it can just get the string once
 
> > `hasProperty` can be used for more concise code instead of `getProperty`
> 
> If the `"buildVST"` property is set to 0, `bool{getProperty("buildVST")}` will return `false`, while `hasProperty("buildVST")` will return `true`. So here, we can't use `hasProperty`.

Oops, right.

> > Probably makes more sense to check `pluginFormats` first to short-circuit most times, since that's where it would be for non-legacy jucer files
> 
> I totally agree! Addressed in [8460bc9](https://github.com/McMartin/FRUT/commit/8460bc9c793a5a5ade961feec30b1103d346f344).
👍

Reply:
> I suggest refactoring into a function which checks both, then it can just get the string once

That's actually what I had in mind, but I couldn't find a solution that was elegant enough. I'll leave the code as-is for now. I have plans to change how `Jucer2Reprojucer` works internally in the future anyway.